src/lsm/manifest_level_fuzz.zig

```diff
@@ -0,0 +1,426 @@
+//! Fuzz ManifestLevel insert_table()/set_snapshot_max()/remove_table*()/iterator().
```

Comment:
We should indicate whether all public methods are covered here, so that the reader doesn't have to work this out.

For example, `All public methods are covered.`